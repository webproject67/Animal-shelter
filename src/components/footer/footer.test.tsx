import * as React from "react";
import * as renderer from "react-test-renderer";
import Footer from "./footer";

it(`Render Footer`, () => {
  const children = <div className="children-component" />;

  const root = renderer.create(<Footer>
    {children}
  </Footer>).toJSON();

  expect(root).toMatchSnapshot();
});