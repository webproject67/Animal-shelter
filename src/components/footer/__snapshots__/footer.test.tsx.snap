// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer 1`] = `
Array [
  <div
    className="children-component"
  />,
  <footer
    className="page-footer"
  >
    <div
      className="page-footer__wrapper"
    >
      <div
        className="page-footer__contacts"
      >
        <b
          className="page-footer__title"
        >
          For questions 
          <br />
           and suggestions
        </b>
        <a
          className="page-footer__email"
          href="mailto:email@shelter.com"
        >
          <svg
            className="page-footer__icon"
          >
            <use
              xlinkHref="img/svg/sprite.svg#envelope"
            />
          </svg>
          email@shelter.com
        </a>
        <a
          className="page-footer__tel"
          href="tel:+136745677554"
        >
          <svg
            className="page-footer__icon"
          >
            <use
              xlinkHref="img/svg/sprite.svg#telephone"
            />
          </svg>
          +13 674 567 75 54
        </a>
      </div>
      <div
        className="page-footer__contacts"
      >
        <b
          className="page-footer__title"
        >
          We are waiting 
          <br />
           for your visit
        </b>
        <button
          className="page-footer__address"
        >
          <svg
            className="page-footer__icon page-footer__icon--pin"
          >
            <use
              xlinkHref="img/svg/sprite.svg#pin"
            />
          </svg>
          Boston, Central Street, 1st 
          <br />
           (Entrance from the store)
        </button>
        <button
          className="page-footer__address"
        >
          <svg
            className="page-footer__icon page-footer__icon--pin"
          >
            <use
              xlinkHref="img/svg/sprite.svg#pin"
            />
          </svg>
          London, South Park, 18st
        </button>
      </div>
    </div>
  </footer>,
]
`;
